README.md

```diff
@@ -61,24 +62,31 @@ Quick legend:
 | Client Lifecycle Events    |  ✔  |     ✔      |
 | Client Tick Events         |  ✔  |     ✔      |
 | Client World Tick Events   |  ✔  |     ✔      |
-| Client Block Entity Events |  ❌  |     ✔      |
-| Client Chunk Events        |  ❌  |     ✔      |
-| Common Lifecycle Events    |  ❌  |     ✔      |
+| Client Block Entity Events |  ❌ |     ✔      |
+| Client Chunk Events        |  ❌ |     ✔      |
+| Client Entity Events       |  ❌ |     ✔      |
+| Common Lifecycle Events    |  ❌ |     ✔      |
 | Server Lifecycle Events    |  ✔  |     ✔      |
 | Server Tick Events         |  ✔  |     ✔      |
 | Server World Load Events   |  ✔  |     ✔      |
 | Server World Tick Events   |  ✔  |     ✔      |
-| Server Block Entity Events |  ❌  |     ✔      |
-| Server Chunk Events        |  ❌  |     ✔      |
+| Server Block Entity Events |  ❌ |     ✔      |
+| Server Chunk Events        |  ❌ |     ✔      |
+| Server Entity Events       |  ❌ |     ✔      |
 
 ### Core - Registry
 
 | Feature                                     | QSL | Fabric API |
 |:--------------------------------------------|:---:|:----------:|
-| Addition Events                             |  ✔  |     ✔      |
-| Addition Events Helper                      |  ✔  |     ❌      |
-| Registry Syncing                            |  ✔  |     ✔      |
-| Registry Syncing - Exclude Specific Entries |  ✔  |     ❌      |
+| Addition Events                             |  ✔  |     ✔     |
+| Addition Events Helper                      |  ✔  |     ❌    |
+| Dynamic Registry Setup Event                |  ✔  |     ✔     |
+| Dynamic Registry Loaded Event               |  ✔  |     ❌    |
+| Synced Dynamic Registry Registration        |  ✔  |     ✔     |
+| Unsynced Dynamic Registry Registration      |  ✔  |     ✔     |
+| Dynamic Registry Flags                      |  ✔  |     ❌    |
+| Registry Syncing                            |  ✔  |     ✔     |
+| Registry Syncing - Exclude Specific Entries |  ✔  |     ❌    |
```

Comment:
I think there is a PR for this or something similar. I would double check the available API for registry stuff.

Reply:
I didn't find anything in Fabric's PRs neither about Flags nor Excluding Specific Entries